pkg/actuators/awsplacementgroup/controller.go

```diff
@@ -5,7 +5,7 @@ import (
 	"fmt"
 
 	"github.com/go-logr/logr"
-	machinev1 "github.com/openshift/api/machine/v1"
+	machinev1 "github.com/openshift/machine-api-provider-aws/api"
```

Comment:
I would suggest adding more folder levels to the import here, I think it should be `pkg/api/machine/v1`
If we don't add the folder levels, it's weird that the import is machine v1 right? Also, we might need other API hacks in the future that this might conflict with, best to have some structure IMO